### DISCUSSION TITLE
Cannot read properties of undefined (reading 'current')

### DISCUSSION THREAD

**Discussion by RigottiG** [preactjs/preact#3541] [Q&A]
I'm trying to use Preact with Next then I install Preact and rewrite a config on next.config.js but I'm facing the following issue when run `yarn start` the page doesn't work, just shows a blank screen with some content. 

![image](https://user-images.githubusercontent.com/13246770/167645383-350f8eaf-2ac8-42ce-a093-c1e71ba09d85.png)

```javascript
  webpack: (config, { dev, isServer }) => {
    if (!dev && !isServer) {
      Object.assign(config.resolve.alias, {
        react: 'preact/compat',
        'react-dom/test-utils': 'preact/test-utils',
        'react-dom': 'preact/compat',
      })
    }

    return config
  },
```

**Comment by 657kbps** [comment]
Check where you use useRef or createRef in your code, the problem could be caused by something like this:

```js
const someRef = useRef()

useEffect(() => {
  console.log(someRef.current)  // will throw an error "Cannot read properties of undefined(reading 'current')" because the current value of someRef is undefined
}, [])

```
